transport/api/src/lib.rs

```diff
@@ -453,8 +453,22 @@ where
         let mut transport_layer =
             HoprSwarm::new((&self.me).into(), discovery_updates, self.my_multiaddresses.clone()).await;
 
-        if let Some(port) = self.cfg.protocol.autonat_port {
+        let autonat_port = self.my_multiaddresses.first().and_then(|addr| {
```

Comment:
There's only one multiaddress coming from the config... so it's always that one, but why the first one if more are available? 

Why not make a list of unique ports that you service and open for all of them?